src/bundle/DependencyInjection/Configuration/Parser/AdminUiForms.php

```diff
@@ -87,6 +88,12 @@ public function addSemanticConfig(NodeBuilder $nodeBuilder)
                                     ->end()
                                 ->end()
                             ->end()
+                            ->arrayNode('meta_fieldgroup_list')
+                                ->performNoDeepMerging()
+                                ->scalarPrototype()->end()
+                                ->info('List of field groups that would be placed under a meta section in a content form')
+                                ->defaultValue(['metadata'])
```

Comment:
Q to @SylvainGuittard if we should do that by default. It'll move existing fields to Meta tab, not sure if this is expected by all customers.

Reply:
Also adding to the questions, what we should do with removing `meta: true` from different fieldtypes like `ibexa_taxonomy_entry_assignment`. PRs are there, but not sure if I should close them now, for reference:
- https://github.com/ibexa/taxonomy/pull/177
- https://github.com/ibexa/product-catalog/pull/868

Reply:
I think we should do it by default. It makes sense with the new layout.